G.A.M.M.A/modpack_addons/G.A.M.M.A. Additional Storylines fixes/gamedata/configs/text/eng/st_ah_quest_ah.xml

```diff
@@ -544,117 +544,117 @@
-        <text>Cold reported that the scout's DA has been deciphered and is waiting to meet me.</text>
+        <text>Cold reported that the scout's PDA has been deciphered and he is waiting to meet me.</text>
     </string>
     <string id="cs_task4_storytask_name">
-        <text>Last Dawn: meet the mercenary Scar</text>
+        <text>Last Dawn: Find the Mercenary Scar</text>
     </string>
     <string id="cs_task4_storytask_descr">
         <text>I managed to figure out where Scar is. Now my task is to find out what he knows about the missing members of Clear Sky.</text>
     </string>
     <string id="cs_task5_storytask_name">
-        <text>Last Dawn: fight off mutants</text>
+        <text>Last Dawn: Fight off the mutants</text>
     </string>
     <string id="cs_task5_storytask_name1">
-        <text>The Last Dawn: talk to Scar.</text>
+        <text>The Last Dawn: Talk to Scar.</text>
     </string>
     <string id="cs_task5_storytask_descr">
-        <text>We are attacked by mutants, we need to fight back.</text>
+        <text>There is a pack of pseudodogs thats picked up my scent, I came here to make a stand. Help me see them off and I'll tell you what you want to know.</text>
     </string>
     <string id="cs_task3_storytask_name">
-        <text>The Last Dawn: find out about Scar.</text>
+        <text>The Last Dawn: Find out about Scar.</text>
     </string>
     <string id="cs_task3_storytask_descr">
-        <text>Need to make inquiries about the mercenary named Scar from Sidorovich.</text>
+        <text>I need to make inquiries about the Mercenary named Scar, that old crook Sdorovich should be able to point me in the right direction.</text>
     </string>
     <string id="cs_task9_storytask_name">
-        <text>The Last Dawn: talk to the Strider</text>
+        <text>The Last Dawn: talk to Strider</text>
     </string>
     <string id="cs_task9_storytask_descr">
-        <text>It is necessary to take the decoder to Strider. His unit is located in the vicinity of Jupiter, at the checkpoint.</text>
+        <text>We have to get the decoder to Strider. His unit set up a small base of opperations in Jupiter, at the old checkpoint South of Yanov Station.</text>
     </string>
     <string id="cs_task11_storytask_name">
-        <text>The Last Dawn: talk to the Strider</text>
+        <text>The Last Dawn: Talk to Strider</text>
     </string>
     <string id="cs_task11_storytask_descr">
-        <text>Scar reported that the surviving CS'es may be in the ranks of the Outcasts of the Monolith. His unit is located in the vicinity of Jupiter, at the checkpoint. It is necessary to find him and collect all the information.</text>
+        <text>Scar reported that the Clear Sky survivors may have joined the Outcasts of the Monolith. Their leader's name is Strider, his unit is located in Jupiter at the old checkpoint South of Yanov Station. I must find him and ask if he knows of the missing Clear Sky stalkers.</text>
```

Comment:
Their leader's name is Strider **and** his unit is located in Jupiter at the old checkpoint South of Yanov Station. I must find him and ask if he knows **about the missing survivors**.